internal/documentation/docs/updates/migrate-v5.md

```diff
@@ -0,0 +1,116 @@
+# Migrate to v5
+
+::: tip Alpha Release Available
+**UI5 CLI 5.0 Alpha is now available for testing! 🎉**
+
+Try the alpha release in your projects via: `npm i --save-dev @ui5/cli@next`  
+Or update your global install via: `npm i --global @ui5/cli@next`
+
+**Note:** This is a pre-release version.
+:::
+
+## Breaking Changes
+
+- **All UI5 CLI Modules require Node.js >= 22.20.0 or >= 24.0.0**
+
+- **@ui5/cli: `ui5 init` defaults to Specification Version 5.0**
+
+- **@ui5/project: New Component Type**
```

Comment:
What is the breaking change w.r.t. the introduction of the new type? The type needs to be actively used (new feature). I don't see any incompatible changes when not actively changing something.